modules/adb-lakehouse-uc/account-principals/README.md

```diff
@@ -0,0 +1,52 @@
+# Unity Catalog terraform blueprints
+
+This module contains Terraform code used to provision a Databricks service principal on account-level.
+
+## Module content
+
+This module can be used to deploy the following:
+
+* A Azure Databricks Service Principal
+
+## How to use
+
+1. Reference this module using one of the different [module source types](https://developer.hashicorp.com/terraform/language/modules/sources)
+2. Add a `variables.tf` with the same content in [variables.tf](variables.tf)
+3. Add a `terraform.tfvars` file and provide values to each defined variable
+4. Add a `output.tf` file.
+5. (Optional) Configure your [remote backend](https://developer.hashicorp.com/terraform/language/settings/backends/azurerm)
+6. Run `terraform init` to initialize terraform and get provider ready.
+7. Run `terraform apply` to create the resources.
```

Comment:
This is not strictly required - it's just best practice, but not related to databricks